src/python/module/smartredis/client.py

```diff
@@ -27,14 +27,16 @@
 import inspect
 import os
 import os.path as osp
+import functools
 
 import numpy as np
 
 from .dataset import Dataset
-from .error import RedisConnectionError, RedisReplyError
 from .smartredisPy import PyClient
-from .util import Dtypes, init_default
+from .util import Dtypes, init_default, exception_handler
 
+from .error import *
+from .smartredisPy import RedisReplyError as RRE
```

Comment:
Lets spell this out, minor thing.. but I like seeing the name. ``RRE`` looks like a python global too much.

Reply:
The full name is a name conflict so I needed an alias. I'll do something less like a global though